src/debugpy/_vendored/pydevd/setup_cython.py

```diff
@@ -196,7 +196,7 @@ def build_extension(dir_name, extension_name, target_pydevd_name, force_cython,
             Extension(
                 "%s%s.%s" % (dir_name, "_ext" if extended else "", target_pydevd_name,),
                 c_files,
-                **kwargs,
+                **kwargs
```

Comment:
It seems having the trailing comma here is not valid on Python 2 (so, this is an unrelated fix for Python 2).